tests/MileageTest.cpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * @file MileageTest.cpp
+ * @brief 走行距離の測定用クラスのテスト
+ * @author molpui0726
+ */
+
+#include <cmath>
+#include <gtest/gtest.h>
+#include "Mileage.h"
+
+namespace etrobocon2025_test {
+  TEST(MileageTest, CalculateMileage)
```

Comment:
下がMinus, ZeroならPlusですかね
```suggestion
  TEST(MileageTest, CalculateMileagePlus)
```